pkg/sfu/receiver.go

```diff
@@ -549,6 +530,9 @@ func (w *WebRTCReceiver) forwardRTP(layer int32) {
 		})
 
 		w.streamTrackerManager.RemoveTracker(layer)
+		if w.isSVC {
+			w.streamTrackerManager.RemoveAllTrackers()
+		}
```

Comment:
Adding this as SVC codecs add trackers as necessary in this routine. Once the SVC stream is gone, all trackers should be removed.